langlib/langlib-test/src/test/java/org/ballerinalang/langlib/test/statements/foreach/ForeachStringTypedBindingPatternsTests.java

```diff
@@ -30,7 +30,7 @@
  *
  * @since 0.985.0
  */
-@Test(enabled = false)
+@Test
 public class ForeachStringTypedBindingPatternsTests {
```

Comment:
Let's also add a negative test in org.ballerinalang.langlib.test.statements.foreach.ForeachNegativeTests#testForeachVarTypeNegative.

For something like
```ballerina
    string str = "foo";

    foreach int s in str {
        
    }
```

we should now get

```
incompatible types: expected 'string:Char', found 'int'
```